C_CPP/Data Structures/Linked List/Intersecting Linked List/readme_IntersectingLL.md

```diff
@@ -0,0 +1,71 @@
+# Intersecting Linked Lists
+
+<p>When two linked lists intersect at a point (node), we refer to them as intersecting linked lists.
+
+Following is an example : </p>
+</br>
+
+```
+h1 - 1 - 2 - 3 - 4
+                  \
+                   5 - 6 - 7 - 8 - 9 - 10 - NULL
+                  /
+            h2 - 3
+
+The 2 linked lists intersect at node 5. 
+```
+</br>
+
+## Intersection
+
+<p> To just check whether the 2 nodes intersect or not, the sufficient condition is that they have the same tail.</br>
+So, we can just check whether the addresses of the two linked lists have the same address or not.
+</p>
+</br>
+
+## Finding node of Intersection
+
+<p> For finding the node at which two linked lists intersect we can compare the addresses of each node of a linked list with the other, but this would give the time complexity as O(m*n), where m and n are the lengths of the individual linked lists.
+</br>
```

Comment:
```suggestion
For finding the node at which two linked lists intersect we can compare the addresses of each node of a linked list with the other, but this would give the time complexity as O(m*n), where m and n are the lengths of the individual linked lists.  
  
```